Jenkinsfile

```diff
@@ -128,6 +91,9 @@ pipeline {
                             context.isStageIncluded()
                         }
                     }
+                    environment {
+                        GENERATE_REQUIREMENTS=1
```

Comment:
Why does this need to be set globally?  Can we add a `generateRequirements` function to the Jenkinsfile, or just invoke `scripts/generate_requirement.sh` directly?

Reply:
This is also a bit confusing, because having this set globally would cause the `fetch_firmare.sh` script to not actually download external artifacts, it would instead just generate `requirements.yaml`.

Reply:
Fixed - now fetching firmware correctly in "internal" build, see https://jenkins.ci.swift-nav.com/blue/organizations/jenkins/swift-nav%2Fpiksi_buildroot/detail/PR-1217/43/pipeline/28.